ci/travis/ci.sh

```diff
@@ -179,6 +178,10 @@ test_python() {
     bazel test --config=ci \
       --build_tests_only $(./scripts/bazel_export_options) \
       --test_env=PYTHONPATH="${PYTHONPATH-}${pathsep}${WORKSPACE_DIR}/python/ray/pickle5_files" \
+      --test_env=USERPROFILE="${USERPROFILE}" \
+      --test_env=CI=1 \
+      --test_env=RAY_CI_POST_WHEEL_TESTS=1 \
+      --test_output=streamed \
```

Comment:
Streaming the output forced tests to not be run in parallel, but it does not seem to increase the time required for CI. Part of PR #23188, repeated here.